src/main/resources/org/sonatype/nexus/ci/iq/IqPolicyEvaluatorWorkflowStep/help-manualAppId.html

```diff
@@ -0,0 +1,18 @@
+<!--
```

Comment:
The name/contents of these help files needs to be reassessed with the switch to hetero_radio.

Reply:
> The name/contents of these help files needs to be reassessed with the switch to hetero_radio.

Changed dd6ae3b

Reply:
Missed some changes